Update spec.js

AleAb25 opened the pull request:
I didn't succeed in re-create the duplicate name folder message as I just couldn't send any keys to the name or description field the second time I've opened the new folder dialog window (no matter how many times and how many different scenarios between them).

AleAb25 committed:
Update spec.js

I didn't succeed in re-create the duplicate name folder message as I just couldn't send any keys to the name or description field the second time I've opened the new folder dialog window (no matter how many times and how many different scenarios between them).

gmandakini commented on starter/spec.js:
```diff
@@ -1,9 +1,195 @@
+            expect(newFolderName.isDisplayed()).toBe(true);
+        });
+
+        // it('should check for a duplicate message', function(){
```
Can this test be uncommented and get it working. Maybe it's actually failing in the next test, because the new folder dialog is not closed?

AleAb25 replied:
I've simplified the test and got it to work.

gmandakini commented on starter/spec.js:
```diff
@@ -1,9 +1,195 @@
+            newFolderName.click().then(function(){
+                threeDots.click().then(function(){
+                    expect(folderMenuOptions.isDisplayed()).toBe(true).then(function(){
+                        deleteFolderOptionButton.click();
```
Can you please add the assertion for the deleted folder message that is displayed to the user.

AleAb25 replied:
Hello, done. I've confirmed that the folder is deleted by checking that the confirmation message is displayed and that it contains the "deleted" keyword.

AleAb25 committed:
Updated delete folder confirmation

I've confirmed that the folder is deleted by checking that the confirmation message is displayed and that it contains the "deleted" keyword.

AleAb25 committed:
Check for duplicate name message

I've simplified the test and got it to work.
